app/controllers/admin/tracks_controller.rb

```diff
@@ -36,4 +36,17 @@ def update_tracks
       end
     end
   end
+
+  def update_offset
+    params[:time][0].each do |id, value|
+      talk = Talk.find(id)
```

Comment:
特に理由が無ければ `update!` で良いかと思いました
```suggestion
      Talk.find(id).update!(
          start_offset: value[:startOffset],
          end_offset: value[:endOffset]
      )
```